__tests__/component/form/CreateCustomerForm.test.tsx

```diff
@@ -50,15 +51,30 @@ const fillAndSubmitForm = async (customer: Customer) => {
     await waitFor(() => fireEvent.click(submitButton))
 }
 
+const mockRequest = (): object => bodySpy.getCalls()[0].args[0]
```

Comment:
It's probably better to simply define this as a (non function) constant in each test body. Please rename to `testRequestBody`.